inference/core/usage.py

```diff
@@ -29,6 +29,9 @@ def trackUsage(endpoint, actor, n=1):
     """
     # count an inference
     try:
+        logger.warn(
```

Comment:
why warn?

Reply:
The default log level is set to warning, so info and debug logs don't get outputted. Setting the default level lower would create a lot of noise. I also tried using the `print` call, but this doesn't format logs in a way that the Cloudwatch-Lambda trigger will pick up (they still get logged, just not picked up by the trigger)

Reply:
there is an option to create custom logs levels which we should do

Reply:
Oh that's a good idea, I will do that

Reply:
Custom logging [doesn't seem to be recommended](https://docs.python.org/3.5/howto/logging.html#custom-levels). In any case, I've been having issues getting it to work. I even tried adding a function to temporarily set the log level lower, then reset it back but that also didn't work. Not sure if there's some configuration I'm missing here

Reply:
I believe this is recommended for our use case.

Maybe we could just to this:
![image](https://github.com/user-attachments/assets/af9f79a6-0ef7-4c7b-9dc6-d5bf93eb72a8)

Reply:
also works with structlog
![image](https://github.com/user-attachments/assets/c1f4ef2a-3484-4714-b94a-c960359356eb)

Reply:
btw - why do we need this logging here?

Reply:
if we need to make sure that the thing work - we should eliminate the bugs - I believe I saw and report problems with `usage.py` - let's start from fixing error handling by logging error context, then fix the proble